tracing-log/src/lib.rs

```diff
@@ -72,8 +70,6 @@
 //! * `trace-logger`: enables an experimental `log` subscriber, deprecated since
 //!   version 0.1.1.
```

Comment:
let's also remove this before shipping an 0.2.0? i'm fine with doing that in this PR, or in a separate one, up to you.

Reply:
done in this PR!